Products/GenericSetup/tool.py

```diff
@@ -921,17 +921,30 @@ def getProfileInfo(self, profile_id):
 
     @security.protected(ManagePortal)
     def getDependenciesForProfile(self, profile_id):
+
+        dependencies = None
+        profile_info = None
+
         if profile_id is None:
-            return ()
+            dependencies = ()
+
         if profile_id.startswith("snapshot-"):
-            return ()
+            dependencies = ()
+
+        profile_info = self.getProfileInfo(profile_id)
+
+        dependencies = profile_info.get('dependencies', ())
+
+            for dependency_id in dependencies:
```

Comment:
Unindent here (this is a `SyntaxError`).

Reply:
My bad, used the web-editor, that produced tabs instead of spaces.
Making them spaces was not recognized as a changement, so added a space somewhere else, to make it commitable. Travis is super-fast, wow.